docs/source/getting-started/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/DecodeUtil.scala

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.quickstart.iou
+
+import com.digitalasset.ledger.api.v1.event.{CreatedEvent, Event}
+import com.digitalasset.ledger.api.v1.transaction.Transaction
+import com.digitalasset.ledger.api.v1.{value => V}
+import com.digitalasset.ledger.client.binding.{Template, ValueDecoder, Primitive => P}
+
+object DecodeUtil {
+  def decodeAllCreated[A <: Template[_]: ValueDecoder](
+      transaction: Transaction): Seq[(P.ContractId[A], A)] =
```

Comment:
Replace throughout this file:

```suggestion
      transaction: Transaction): Seq[Contract[A]] =
```